lib/lightstep/span.rb

```diff
@@ -92,18 +92,16 @@ def get_baggage_item(key)
     def log(event: nil, timestamp: Time.now, **fields)
       return unless tracer.enabled?
 
+      fields = {} if fields.nil?
+      unless event.nil?
+	fields[:event] = event.to_s
+      end
       record = {
-        runtime_guid: tracer.guid,
```

Comment:
it was needed many many :moon:s ago but no longer... could have been omitted from day one. (it's implied transitively by the associated span)